specs/decisionmatrix/aws.yaml

```diff
@@ -0,0 +1,3620 @@
+rows:
```

Comment:
in the generated file, add a comment to the top that this file is generated automatically using the the make rule and do not edit manually